edx_repo_tools/release/tag_release.py

```diff
@@ -88,6 +90,17 @@ def repo_matches(repo, pattern):
         fnmatch.fnmatch(repo.name, pattern)
     )
 
+
+def keep_only_repos(repos, keep_repos):
```

Comment:
It's a minor thing, but I find "include" is a more natural verb than "keep" for this sort of behavior.  Would it be a pain to update throughout?